admin_manual/installation/nginx.rst

```diff
@@ -71,9 +69,14 @@ webroot of your nginx installation. In this example it is
       add_header X-Robots-Tag none;
       add_header X-Download-Options noopen;
       add_header X-Permitted-Cross-Domain-Policies none;
+      fastcgi_hide_header X-Content-Type-Options;
+      fastcgi_hide_header X-XSS-Protection;
```

Comment:
We should not remove them by default, because then it could be that those are not set by the web server as well and causing a security problem.